ddtrace/constants.py

```diff
@@ -1,3 +1,5 @@
+# This key allows the agent to ignore the sampling_priority set on a span. This
```

Comment:
When you say sampling_priority you mean `_sampling_priority_v1` which is set on the context? Could we clarify the comment?